src/test/scala/symsim/concrete/ConcreteSarsaIsSarsaSpec.scala

```diff
@@ -11,7 +11,7 @@ class ConcreteSarsaIsSarsaSpec extends SymSimSpec {
       CarFiniteState,
       CarAction
     ] (
-      agent = Car,
```

Comment:
This changes are not needed for this commit (but they do not harm much either) - they are related to testing SARSA not this agent. We can keep them (easier than revert).